packages/ibm-products/src/components/FilterPanel/FilterPanelCheckbox/FilterPanelCheckbox.test.js

```diff
@@ -44,6 +45,14 @@ describe(componentName, () => {
     expect(screen.getByRole('checkbox')).toBeInTheDocument();
   });
 
+  it('renders a count', async () => {
+    const filterPanelLabelClass = `${pkg.prefix}--filter-panel-label`;
+    const { container } = renderComponent({ count: count });
```

Comment:
Just semantics but just passing count feels a bit cleaner
```suggestion
    const { container } = renderComponent({ count });
```